pkg/opni/commands/metrics.go

```diff
@@ -14,6 +14,7 @@ func BuildMetricsCmd() *cobra.Command {
 
 	cmd.AddCommand(BuildCortexAdminCmd())
 	cmd.AddCommand(BuildCortexOpsCmd())
+	cmd.AddCommand(BuildMetricsConfigCmd())
```

Comment:
would it make more sense to have capabilty configuration commands under the `opni capabilty` command? Feels out of scope

Reply:
Plugin-specific commands are grouped together so that they can be compiled out using build tags. The current structure keeps the core commands separated. I agree it's not the most obvious structure. Refactoring would be out of scope for this PR but we can definitely take a look at updating this

Reply:
Ok works for me